build/Makefile.pf.AIX

```diff
@@ -19,7 +19,7 @@ C_WARN_FLAGS=-qformat -qhalt=w -qflag=w:w -qsuppress=1540-2864
 # Common compiler flags (sent to compile command and link commands)
 # (-qthreaded is implied by using the _r version of xlc++)
 #64bit: COMMONFLAGS=-q64 -qlanglvl=strict98 -qrtti -qpic
-COMMONFLAGS=-q32 -qlanglvl=strict98 -qrtti -qpic -qextchk
+COMMONFLAGS=-q32 -qlanglvl=strict98 -qrtti -qpic -qextchk -qlanglvl=anonstruct
```

Comment:
Yes the cdli_entuser.lncent.h has an anonymous structure and hence I had to apply the flag anonstruct. I am in process of running the pbuild, once done I will check-in this.
Thanks for the review.